Oqtane.Client/Modules/Admin/Register/Index.razor

```diff
@@ -94,4 +97,16 @@
     {
         NavigationManager.NavigateTo(NavigateUrl(""));
     }
+}
+    private void IsValidEmail()
```

Comment:
make bool function and call it directly
private bool IsEmailValid(string email)  {}